client/src/components/QA/QACardAnswers.jsx

```diff
@@ -62,9 +62,11 @@ const QACardAnswers = ({ currentAnswers }) => {
         }}><b><span className="qa-load">[LOAD MORE ANSWERS]</span></b></button>
```

Comment:
I don't see how this button is conditionally being shown. It seems like it will always be shown, to me.